src/fluka/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ find_package(Fluka REQUIRED)
 set(SRC_FILES fluka_funcs.cpp)
 set(PUB_HEADERS fluka_funcs.h)
 
-set(LINK_LIBS ${FLUKA_LIBRARIES} dagmc pyne_dagmc uwuw)
+set(LINK_LIBS ${FLUKA_LIBRARIES} dagmc pyne_dagmc uwuw quadmath)
```

Comment:
This is the only line remaining in this PR that I think is still needed. You cannot build a static fludag executable without this change.

Reply:
I'm still going to close this PR though. That one line can be its own PR (or we can just ignore it and leave it broken)